PropertyInformationApi/Startup.cs

```diff
@@ -110,6 +110,7 @@ public static void ConfigureServices(IServiceCollection services)
             ConfigureDbContext(services);
             RegisterGateways(services);
             RegisterUseCases(services);
+            services.AddScoped<IEntityMapper, EntityMapper>();
```

Comment:
You could probably keep it as a singleton like the original source?
If you take a look at line 120, it's already added as a singleton there.
You could just call the method on line 116 here instead, as the mapper has to be configured also.

Reply:
Didn't catch that, cheers
I've amended the commit.
The `ConfigurePropertyFactory()` was never called before was it? Or did I miss something?

Reply:
I edited it out when I was converting it.